astropy/stats/info_theory.py

```diff
@@ -162,7 +162,7 @@ def bayesian_info_criterion_lsq(ssr, n_params, n_samples):
     >>> y = 3 * np.exp(-0.5 * (x - 1.3)**2 / 0.8**2)
     >>> y += np.random.normal(0., 0.2, x.shape)
     >>> # Fit the data using a Box model
-    >>> t_init = models.Trapezoid1D(amplitude=1., x_0=0., width=1., slope=0.5)
+    >>> t_init = models.Trapezoid1D(amplitude=1., x_0=0., width=1., slope=0.5, bounds=["x_0": (-1., 1.)])
```

Comment:
`bounds` should be a dictionary with curly brackets. Same as the example below. This is causing `doctest` failures.

Reply:
@pllim I made the changes accordingly.